plugin-server/src/worker/vm/lazy.ts

```diff
@@ -26,47 +26,87 @@ export class LazyPluginVM {
     resolveInternalVm!: Promise<PluginConfigVMResponse | null>
     totalInitAttemptsCounter: number
     initRetryTimeout: NodeJS.Timeout | null
+    ready: boolean
+    responseVar: string | null
 
     constructor() {
         this.totalInitAttemptsCounter = 0
         this.initRetryTimeout = null
+        this.ready = false
+        this.responseVar = null
```

Comment:
Nit: the name `responseVar` makes sense inside `vm.ts`, but not if I'm looking at this constructor. `vmResponseVariable`? non-blocking obviously...